src/matches/matches.controller.ts

```diff
@@ -47,15 +48,17 @@ export class MatchesController {
   @ApiOperation({ summary: 'Create match' })
   async createMatch(@Param() params: LeagueParams, @Body() dto: CreateMatchDto): Promise<Match> {
     const { leagueIdx, homeTeamIdx } = await this.getUserReadableKeyParams(dto.homeTeamId);
-    return this.matchesService.createMatch(params.leagueId, dto, leagueIdx, homeTeamIdx);
+    const { phoneNumber } = await this.usersService.getById(dto.observerId)
```

Comment:
I'd change it to a whole User object named _observer_ because in the future we might also be retrieving _referee_ in here so we can't have two _phoneNumber_ variables.

Reply:
But we may have observerPhoneNumber and refereePhoneNumber
Maybe I'll change it to observerPhoneNumber?

Reply:
Ok